src/easydiffraction/analysis/calculators/calculator_crysfml.py

```diff
@@ -182,16 +184,22 @@ def _convert_experiment_to_dict(
 
         exp_dict = {
             'NPD': {
-                '_diffrn_radiation_probe': expt_type.radiation_probe.value if expt_type else 'neutron',
-                '_diffrn_radiation_wavelength': instrument.setup_wavelength.value if instrument else 1.0,
+                '_diffrn_radiation_probe': expt_type.radiation_probe.value
+                if expt_type
+                else 'neutron',
+                '_diffrn_radiation_wavelength': instrument.setup_wavelength.value
+                if instrument
+                else 1.0,
                 '_pd_instr_resolution_u': peak.broad_gauss_u.value if peak else 0.0,
                 '_pd_instr_resolution_v': peak.broad_gauss_v.value if peak else 0.0,
                 '_pd_instr_resolution_w': peak.broad_gauss_w.value if peak else 0.0,
                 '_pd_instr_resolution_x': peak.broad_lorentz_x.value if peak else 0.0,
                 '_pd_instr_resolution_y': peak.broad_lorentz_y.value if peak else 0.0,
-                # "_pd_instr_reflex_s_l": peak_asymm.s_l.value if peak_asymm else 0.0,
-                # "_pd_instr_reflex_d_l": peak_asymm.d_l.value if peak_asymm else 0.0,
-                '_pd_meas_2theta_offset': instrument.calib_twotheta_offset.value if instrument else 0.0,
+                # "_pd_instr_reflex_s_l": peak_asymm.s_l.value if ...
+                # "_pd_instr_reflex_d_l": peak_asymm.d_l.value if ...
```

Comment:
Why the commented out code is changed? This is no longer a commented out code but an actual comment then, so maybe some extra description would be needed to tell why it is here.